<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
<html><body>
<h1 id="tutorial">
<i class="icon fa-solid fa-feather-pointed"></i>Tutorial</h1>

<div class="search-box">
  <input type="text" class="search-input" oninput="onSearchInput(this)" placeholder="Search items on this page">
  <button disabled data-tooltip="Clear search" aria-label="clear search" onclick="onSearchClear()">
    <i class="icon fa-solid fa-magnifying-glass"></i>
  </button>
</div>

<div class="tags" data-link="https://feliciagomezlab.github.io/Gomez-Lab.github.io/tutorial/">
    
      <a href="https://feliciagomezlab.github.io/Gomez-Lab.github.io/tutorial/?search=%22tag:%20tutorial%22" class="tag" data-tooltip='Show items with the tag "tutorial"'>
        tutorial
      </a>
    
      <a href="https://feliciagomezlab.github.io/Gomez-Lab.github.io/tutorial/?search=%22tag:%20galaxy%22" class="tag" data-tooltip='Show items with the tag "galaxy"'>
        galaxy
      </a>
    
      <a href="https://feliciagomezlab.github.io/Gomez-Lab.github.io/tutorial/?search=%22tag:%20variant-calling%22" class="tag" data-tooltip='Show items with the tag "variant-calling"'>
        variant-calling
      </a>
    
  </div>

<div class="search-info"></div>

<div class="post-excerpt-container">
  <div class="post-excerpt">
    
    
    

    

    <div class="post-excerpt-text">
      <a href="/Gomez-Lab.github.io/2025/03/10/Galaxy.html">Galaxy Pipeline</a>

      <div class="post-info">
  
    
    
      <span data-tooltip="Author">
        <i class="icon fa-solid fa-feather-pointed"></i>
        <span></span>
      </span>
    
  

  
  

  
    <span data-tooltip="Originally published on">
      <i class="icon fa-regular fa-calendar"></i>
      <span>March 10, 2025</span>
    </span>
  

  
    <span data-tooltip="Last updated on">
      <i class="icon fa-solid fa-clock-rotate-left"></i>
      <span>April 11, 2025</span>
    </span>
  
</div>


  


  <div class="tags" data-link="https://feliciagomezlab.github.io/Gomez-Lab.github.io/blog">
    
      <a href="https://feliciagomezlab.github.io/Gomez-Lab.github.io/blog?search=%22tag:%20tutorial%22" class="tag" data-tooltip='Show items with the tag "tutorial"'>
        tutorial
      </a>
    
      <a href="https://feliciagomezlab.github.io/Gomez-Lab.github.io/blog?search=%22tag:%20galaxy%22" class="tag" data-tooltip='Show items with the tag "galaxy"'>
        galaxy
      </a>
    
      <a href="https://feliciagomezlab.github.io/Gomez-Lab.github.io/blog?search=%22tag:%20variant-calling%22" class="tag" data-tooltip='Show items with the tag "variant-calling"'>
        variant-calling
      </a>
    
  </div>





      
      
      <p data-search="This guide provides a flexible workflow for analyzing next generation sequencing  NGS  data using Galaxy  The tutorial walks you through each step of the process  from setting up your Galaxy environment to interpreting results  ensuring a comprehensive approach to variant analysis  A practice exercise is available to reinforce learning and provide hands on experience for your project Practice Example  Download the sample dataset from Figshare  Click to Download Dataset  Access the example assessment form at  Practice Exercise  Follow the steps provided in this tutorial to analyze a sample dataset   Answer the questions based on your findings and submit your responses Tutorial Sections  General Prerequisites  1  Set Up Your Galaxy Environment  2  Understand Your Sequencing Data  3  Align Reads to a Reference Genome  4  Call Variants  5  Annotate Variants  6  Visualize Variants  7  Analyze and Interpret ResultsGeneral Prerequisites  Objective  Analyze NGS data to identify and interpret variants for  SPECIFIC PURPOSE  e g   clinical reporting  research discovery    Input Data           Paired end FASTQ files   SAMPLE NAME  R1 fq   SAMPLE NAME  R2 fq      BED file   TARGET REGIONS  bed  defines regions of interest         Tools Needed           Galaxy  Register  no download needed       IGV  Download required       wANNOVAR  Use institutional email       VEP  Web based         Hardware  Computer with 100MB 1GB storage  large or dual monitor setup recommended Set Up Your Galaxy EnvironmentGoal  Establish a workspace for data analysis Steps   Visit https   usegalaxy org  and click  Login or Register  to create an account  use Firefox  Chrome  or Safari    Upload or request your instructor collaborator to share the dataset  e g    SAMPLE NAME  R1 fq   SAMPLE NAME  R2 fq   TARGET REGIONS  bed  within Galaxy   Access shared data            History  panel  right   gt   Histories Shared with Me       Select shared history  gt   View   gt   Import History   gt   Start Using This History       Customization  Replace  SAMPLE NAME   e g   TumorSample1  and  TARGET REGIONS   e g   CancerPanel bed  Back to Tutorial SectionsUnderstand Your Sequencing DataGoal  Assess the quality and structure of raw sequencing data Steps   View FASTQ Data           In  History   click the  eye  icon next to  SAMPLE NAME  R1 fq       Examine FASTQ format                         Record run number  e g   585                            Interpret quality scores using ASCII Phred table  e g    5    1  error   B    0 05  error                                              Run Quality Control           Tools  gt   FASTQ Quality Control   gt   FastQC Read Quality Reports       Input   SAMPLE NAME  R1 fq  gt   Run Tool   Repeat for  SAMPLE NAME  R2 fq       View  webpage  output and record       Total sequences      Sequences flagged as poor quality      Sequence length       GC content      Assess quality metrics for concerns  e g   low base quality        Customization  Adjust quality thresholds based on project needs Back to Tutorial SectionsAlign Reads to a Reference GenomeGoal  Map sequencing reads to a reference genome Steps   Align with BWA MEM           Tools  gt   Mapping   gt   Map with BWA MEM   Version 0 7 17 1 or latest       Settings       Reference genome   REFERENCE GENOME   e g   Human hg19       Paired end  Yes      FASTQ files   SAMPLE NAME  R1 fq   SAMPLE NAME  R2 fq      Run Tool  5 15 min   Output   SAMPLE NAME  bam      Note size difference  e g   600MB vs  1GB FASTQ due to compression          Sort the BAM File           Tools  gt   Genomics ToolKits   gt   Picard   gt   SortSam   Version 2 18 2 1       Input   SAMPLE NAME  bam      Sort order   Coordinate       Validation stringency   Lenient       Run Tool        Mark Duplicates           Tools  gt   Picard   gt   MarkDuplicates   Version 2 18 2 1       Input  Sorted  SAMPLE NAME  bam      Run Tool  Outputs  Marked BAM  metrics file      View metrics  e g     PCR duplicates          Remove Duplicates           Tools  gt   SAM BAM   gt   RmDup   Version 2 0 1       Input  Original  SAMPLE NAME  bam      Paired end  Yes      Run Tool  Compare sizes  e g     reduction vs    duplicates        Alternative Approach  Running Without Removing DuplicatesWhile removing duplicates is a common practice to reduce PCR artifacts  in some cases  it might be beneficial to retain duplicates   Low Input DNA  If the sample has a low DNA input  duplicate removal may significantly reduce coverage  potentially affecting variant detection   Unique Molecular Identifiers  UMIs   If UMIs were used  duplicates can be tracked and distinguished  making removal unnecessary   Somatic Variant Calling  Retaining duplicates may help in detecting low frequency somatic mutations in cancer studies   Amplicon Based Sequencing  In targeted sequencing approaches  e g   for viral or bacterial genomes   duplicates may represent true coverage rather than PCR artifacts To run the pipeline without removing duplicates  simply skip the  Remove Duplicates  step and proceed with variant calling as usual Back to Tutorial SectionsCall VariantsGoal  Identify genetic variants from aligned reads Steps   VarScan Workflow           Tools  gt   SAM BAM   gt   Samtools mpileup   Version 2 1 4       Input  Duplicate removed  SAMPLE NAME  bam      Reference   REFERENCE GENOME       Run Tool   Output  Pileup file      Tools  gt   Variant Calling   gt   VarScan   Version 2 4 2       Input  Pileup file      Settings       Min read depth  20      Min supporting reads  4      Min base quality  20      Min variant frequency  0 03      Ignore  gt 90  strand bias  Yes      Run Tool   Output   SAMPLE NAME  VarScan vcf        FreeBayes Workflow           Tools  gt   Variant Calling   gt   FreeBayes   Version 1 1 0 46 0       Input  Duplicate removed  SAMPLE NAME  bam      Reference   REFERENCE GENOME       Parameters  Simple diploid calling  Min coverage  10      Run Tool   Output   SAMPLE NAME  FreeBayes vcf        Limit to Target Regions           Tools  gt   VCF BCF   gt   VCF BEDintersect   Version 1 0 0       Inputs  Each VCF    TARGET REGIONS  bed      Run Tool  twice   Outputs  Filtered VCFs        Compare Variant Callers           Click VCF files in History to view line counts       Record pre post BED intersect counts  e g   VarScan  9 535   48        Customization  Add tools  e g   GATK  or adjust thresholds Back to Tutorial SectionsAnnotate VariantsGoal  Add functional and clinical context to variants Steps   wANNOVAR Annotation           Download filtered VCFs  e g    SAMPLE NAME  VarScan bed vcf  via  disk  icon       Visit http   wannovar wglab org   upload VCF  max 2 runs  use  YOUR EMAIL         Download  genome  CSV       In Excel       Sort by gnomad211 exome AF      Filter ExonicFunc refGeneWithVer  exclude  Synonymous SNV        Filter Func refGeneWithVer  exclude  intron    UTR3          VEP Annotation           Visit http   grch37 ensembl org Homo sapiens Tools VEP      Input  Paste variant  e g   chr7 55221822 55221822 C T  or upload VCF      Run and review annotations         Compare Results           Identify shared and unique variants  e g   TP53 hits        Customization  Use SnpEff or filter by ClinVar Back to Tutorial SectionsVisualize VariantsGoal  Inspect alignments and variants visually Steps   Slice BAM File           Tools  gt   SAM BAM   gt   Slice BAM by genomic regions       Input  Original  SAMPLE NAME  bam    TARGET REGIONS  bed      Run Tool   Download  bam and  bai        View in IGV           Install IGV  load  REFERENCE GENOME   e g   hg19       Load  bam and  bai  same name folder       Zoom to variant  e g   7 140434574       Sort reads by base  right click  gt   Group by base        Check for artifacts  e g   strand bias        Back to Tutorial SectionsAnalyze and Interpret ResultsGoal  Generate insights for  SPECIFIC PURPOSE  Steps   Apply pipeline to your dataset  e g    YOUR SAMPLE  R1 fq    Identify key variants  e g   somatic  actionable    Compare with external data  e g   COSMIC  clinical reports    Document in a report presentation Customization  Focus on your goal  e g   cancer  rare disease  Back to Tutorial SectionsFor further assistance  adapt this pipeline and contact Felicia Gomez ">
        

This guide provides a flexible workflow for analyzing next-generation sequencing (NGS) data using Galaxy. The tutorial walks you through each step of the process, from setting up your Galaxy environment to interpreting results, ensuring a comprehensive approach to variant analysis. A practice exercise is available to reinforce learning and provide hands-on experience for your project.


      </p>
    </div>
  </div>
</div>

</body></html>
